scipy/signal/tests/test_filter_design.py

```diff
@@ -827,6 +827,15 @@ def test_nyquist(self):
         assert_array_almost_equal(w1, w2)
         assert_array_almost_equal(h1, h2)
 
+    def test_fs_validation(self):
+        with assert_raises(ValueError) as e:
```

Comment:
better / more modern and readable form would be something like:
```suggestion
        with pytest.raises(ValueError, match='Sampling.*single scalar'):
```
and no need for the `assert_` later

Reply:
Thanks for pointing me to this syntax. It's integrated now.